minecraft-server/minecraftd.sh

```diff
@@ -118,9 +117,9 @@ idle_server_daemon() {
 		# Retry in ${CHECK_PLAYER_TIME} seconds
 		sleep ${CHECK_PLAYER_TIME}
 
-		if screen -S "${SESSION_NAME}" -Q select . > /dev/null; then
+		if tmux -L "${SESSION_NAME}" select-window -t0:0 2> /dev/null; then
 			# Game server is up and running
-			if [[ "$(screen -S "${SESSION_NAME}" -ls | sed -n "s/.*${SESSION_NAME}\s\+//gp")" == "(Attached)" ]]; then
+			if [[ "$(tmux -L "${SESSION_NAME}" -ls | sed -n "s/.*${SESSION_NAME}\s\+//gp")" == "(Attached)" ]]; then
```

Comment:
`-ls` is not a valid tmux flag, we want to use plain `ls` instead.

Reply:
Furthermore, we don't need any of this `regex` matching with tmux: `[[ $(tmux -L "${SESSION_NAME}" list-sessions -F "#{?session_attached,attached,}") =~ "attached" ]]` should work.

Reply:
I think I found a cleaner way to do that by directly fetching the list of clients connected to that session